doc/storageprofile.md

```diff
@@ -0,0 +1,198 @@
+# Storage Profiles
+
+## Introduction
+
+Storage Profile is the resource that  serves the information about recommended parameters for PVC - right now the accessMode and volumeMode.
+This can be used by CDI controllers when creating a PVC for DV. That way the DataVolume can be simplified and if the properties are missing,
+defaults can be applied from the StorageProfile.
+
+CDI provides a collection of Storage Profiles with default recommended values for some well known backends. 
+If the storage provisioner defined in storage class does not have defaults configured in CDI the resulting StorageProfile 
+has empty `claimPropertySets`.
+
+Storage profile maps 1:1 to StorageClass.
+
+```yaml
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: StorageProfile
+metadata: 
+  name: sample-storage-profile
+spec: 
+  claimPropertySets: 
+  - accessModes:
+    - ReadWriteMany
+    volumeMode: 
+      Filesystem
+status:
+    storageClass: <string>
+    provisioner: <string>
+    claimPropertySets: 
+    - accessModes: array of [ReadWriteMany|ReadWriteOnce|ReadOnlyMany]
+      volumeMode: [Filesystem|Block]
+```
+
+
+## Handling the DV with defaults from Storage Profiles 
+
+The example uses the `hpp` (`kubevirt.io/hostpath-provisioner`) as the storage provisioner.
+For brevity some fields managed by kubernetes, like managedFields or creationTimestamp, were removed from output. 
+
+1. Given the `hostpath-provisioner` StorageClass, CDI creates a `hostpath-provisioner` StorageProfile
+
+`k get sc hostpath-provisioner -o yaml`
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+  name: hostpath-provisioner
+provisioner: kubevirt.io/hostpath-provisioner
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer
+```
+
+`k get storageprofile hostpath-provisioner -o yaml`
+```yaml
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: StorageProfile
+metadata:
+  labels:
+    app: containerized-data-importer
+    cdi.kubevirt.io: ""
+  name: hostpath-provisioner
+  ownerReferences: 
+    ...
+spec: {}
+status:
+  claimPropertySets:
+  - accessModes:
```

Comment:
An alert user will notice this is a list.  We should explain that multiple claim property sets can be specified but currently CDI will ignore all but the first one.

Reply:
now explained on line 37